pkgs/games/space-cadet-pinball/default.nix

```diff
@@ -38,15 +38,21 @@ stdenv.mkDerivation rec {
     SDL2_mixer
     cmake
     makeWrapper
+    Cocoa
   ];
 
-  postInstall = ''
+  preBuild = ''
     mkdir -p $out/lib/SpaceCadetPinball
+    mkdir -p $out/bin
+  '';
+
+  installPhase = ''
     install ${assets}/*.{DAT,DOC,MID,BMP,INF} ${assets}/Sounds/*.WAV $out/lib/SpaceCadetPinball
 
     # Assets are loaded from the directory of the program is stored in
     # https://github.com/k4zmu2a/SpaceCadetPinball/blob/de13d4e326b2dfa8e6dfb59815c0a8b9657f942d/SpaceCadetPinball/winmain.cpp#L119
-    mv $out/bin/SpaceCadetPinball $out/lib/SpaceCadetPinball
+    cp ../bin/SpaceCadetPinball $out/bin
+    cp $out/bin/SpaceCadetPinball $out/lib/SpaceCadetPinball
     makeWrapper $out/lib/SpaceCadetPinball/SpaceCadetPinball $out/bin/SpaceCadetPinball
```

Comment:
I know its already merged but im curious, how is this different from what was previously done?

Reply:
For some reason it wasn't seeing the output binary at `../bin/` with the old installPhase.

Reply:
Ahh, I found it, the cmakelists.txt file has specifically omitted installation on darwin 

https://github.com/k4zmu2a/SpaceCadetPinball/blob/0f88e43ba261bc21fa5c3ef9d44969a2a079d0de/CMakeLists.txt#L221

Perhaps you can just patch that out instead of manually writing the whole install phase using
```
sed -i 's|AND NOT APPLE||' CMakeLists.txt
```
I dont know if macos uses the other files, but they cant hurt. Another option could be to write an install phase specifically for macos while leaving the original one intact. I think we should keep the original one as it automatically installs many other files all of which are useful for linux users.